site/docs/foundations/size.mdx

```diff
@@ -117,6 +117,6 @@ Unlike all other sizes, `--salt-size-border` is a fixed dimension regardless of
 
 <Callout title="Have some feedback?">
 
-We're still in the process of rolling out the Salt Foundations, and we'd appreciate your thoughts and feedback. Please [contact us](../support-and-contributions/index) if you have any comments or suggestions.
+We're still rolling out the Salt Foundations, and we'd appreciate your thoughts and feedback. Please [contact us](../support-and-contributions/index) if you have any comments or suggestions.
```

Comment:
removing loses some intent here?

Reply:
It is indeed :-) Thanks.

I'll go ahead and commit that suggestion.

Reply:
Agree that we should replace with this suggestion.